### ISSUE TITLE
Figure out a way to organize or archive home page banners in Wagtail

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#1737]
The purpose of this issue is to figure out how to organize and archive home page banner announcements in Wagtail.

Beyond the government shutdown emergency banner, we are using the regular home page banner announcements to alert people to Commission meetings, audit hearings, NPRMs, conference registration deadlines, RAD/EFO open late on FEC reporting due dates, and other short term items of importance.

 Some banners need to be re-used (Commission meetings and report deadlines), so it makes sense to have them easily accessible. Some banners don't need to be re-used (e.g., Arizona conference deadline, NPRM deadlines, the original one done for the January 2018 government shutdown), but we most likely need to keep them for archival purposes, and it's helpful to have the language used handy in case we need it in the future for something similar.

Each banner item seems to bring its own discussion and tweaks, and as a result, it's getting messy in the home folder for Wagtail (as that's where banners live in Wagtail). Right now, we've got seven banners in there, and more to come. We need to figure out if there's a way to set up a folder or some other place to store the ones we don't need to re-use, in order to keep Wagtail's home folder relatively clutter-free.

**Comment by dorothyyeager** [comment]
During design sync today @patphongs mentioned an idea she has for organizing the banners that she'd like to test out first on feature or dev. Basically the idea is to create a draft page called banners in the home section, and then move all the banners into being child pages of that new page. I'm happy to help create this and test this out when we're ready to move this out.

**Comment by dorothyyeager** [comment]
I tested out the idea of creating a draft page in feature while testing other features of Wagtail 2.0. It worked great, I moved the banners that were in feature into child pages of the new draft pages, and the home section of Wagtail is cleaned up nicely (at least in feature).

@AmyKort @patphongs @llienfec @johnnyporkchops (aka the other people who get into the banners): This is not the most pressing thing, but perhaps after the internet communications disclaimer deadline passes, in sprint 6.1, I could tackle banner clean up in Wagtail in the prod portion. I'll close this issue and make an implementation issue if you'd like.
